### PR TITLE
Copying across the sprints page from 2016

### PR DESCRIPTION
**Pull request by alexwlchan**
Picking up Daniele’s suggestion from Slack.

### CODE REVIEW

File: pages/what-are-sprints.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+## But I’m not actually a very good Python programmer.
+That’s OK too, neither am I and neither is everyone else. You truly don’t need to be very good Python programmer. In fact, you barely need to be a programmer at all, because even the most novice sprinter can make a very valuable contribution by helping improve documentation, for example.
+
+## I’d be worried about holding up everyone else though.
+It simply doesn’t work like that. For one thing, one of the main points of a sprint is to encourage and help new people. You’ll be able to work at your own pace, learning the things you need to learn as you go along, and you’ll be supported by other people.
+
+## Do I have to stay for a whole sprint?
+Not at all. Drop in just for a bit, if you just want to see what it’s like. There'll be plenty of other things on the code day too, such as [clinics and workshops](/code-day/), so don’t miss them. Other people will be dropping in and out of sprints as well.
```
Comment (alexwlchan) [inline pages/what-are-sprints.md]: Are we doing clinics/workshops this year? Do we need a page for those too?

Comment (evildmp) [reply pages/what-are-sprints.md]: I expect we will, but we don't have anything timetabled yet.

I think we can simply drop that second-last sentence for now @alexwlchan .

Comment (alexwlchan) [reply pages/what-are-sprints.md]: 👍

**Comment by inglesp** [comment]
Thanks both.

**Pull request merged**